src/AgoraGovernor.sol

```diff
@@ -100,10 +109,7 @@ contract AgoraGovernor is
     mapping(address module => bool approved) public approvedModules;
 
     modifier onlyAdminOrTimelock() {
-        require(
-            msg.sender == admin || msg.sender == timelock(),
-            "Only the admin or the governor timelock can call this function"
-        );
+        if (msg.sender != admin && msg.sender != timelock()) revert NotAdminOrTimelock();
```

Comment:
realized we should be using `msgSender()` instead of raw sender
```suggestion
        address sender = _msgSender();
        if (sender != admin && sender != timelock()) revert NotAdminOrTimelock();
```